### DISCUSSION TITLE
How do I move a VM from one folder to another?

### DISCUSSION THREAD

**Discussion by MatthewLStevens** [utmapp/UTM#5694] [macOS]
Update with more info:

If I move my MacOS___.iso file and my MacOS___.utm file to a different location, I get the "Failed to access data from shortcut" error message. However, although I thought the problem was in the QEMU arguments, it seems that the arguments are updated to reflect the new location. But the VM won't open in the new location. (If I move it back, it opens again, although I need to reload the .iso location).

What do I need to do to be able to run the VM from my preferred location (and not the /Downloads folder)?

……………………………………

I've got my MacOS 10.13 VM working, with thanks to this community. I want to move it from its current location (/Downloads) to another location (e.g. /Documents/VMs, or whatever you recommend). I've found that I cannot simply move the .utm and .iso files, as UTM says it can't find them. I see that the locations are stored in the "QEMU Arguments" file. Can I:
- change some settings somwhere in UTM? Or
- edit the Arguments file? If the latter, where is the file stored?

In short, how can I move the .utm and .iso files so that the VM still loads?

**Comment by MatthewLStevens** [comment]
I solved the problem myself by changing something in the VM's settings; I just wish I knew which one!

**Reply by martin0258** [reply]
@MatthewLStevens Which specific setting did you change in the VM's configurations that resolved the problem?

**Reply by MatthewLStevens** [reply]
The short answer is that I wish I knew. I was not systematically changing settings, more just randomly clicking on things, and suddenly it worked. My recollection is that I re-selected the same source files etc. and then pressed Save.
